integration-tests/applyPatches.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eo pipefail
+
+REACT_NATIVE_VERSION=$1
+
+for filename in ../patches/rn-$1/*.patch; do
```

Comment:
I guess that patches represent part of our installation steps as well as changes required to run integration tests on Android simulator. Do we want to have them separately defined for every RN version? Maybe it's better to have set of common patches and RN version specific patches if something is actually broken on their part?

Otherwise changes to installation steps as well as `on-device-disagnose` app might be troublesome.

Reply:
👍